### ISSUE TITLE
import_module from other folder than lib?

### ISSUE THREAD

**Issue by Moritz030** [mrkn/pycall.rb#152]
hello,
is it possible to import a python module from somewhere else than the python lib folder (for example /usr/lib/python3.8/)

**Comment by mrkn** [comment]
@Moritz030 Yes, you can load all the modules that can be found in directories that are contained by `sys.path`.  This is the same manner as Python.
